pyladies_cz.py

```diff
@@ -238,7 +238,7 @@ def read_meetups_yaml(filename):
         meetup['parallel_runs'].append(meetup)
         previous = meetup
 
-    return data
+    return data[::-1]
```

Comment:
Tohle se neučí na začátečnickém kurzu, a blbě se to googlí. Jaký mács názor na `list(reversed(data))` ?

Reply:
Opraveno